README.md

```diff
@@ -125,6 +125,8 @@ and the same test will produce logs:
 step User is logged in as a registered user
 ```
 
+In step implementation you will access matched values using the name of the group, i.e. `match["aRegisteredUser"]`. You can access all matched values (including matched by unnamed groups) by their index, starting from 1, i.e. `match["1"]` (note that key is still a string). So you can have more than one named group and you can mix them with unnamed groups.
```

Comment:
Would it be nice (in a later pull request) for this to not be a dictionary, but by a custom type which allowed strings as named matches, and integers to get unnamed ones?

Reply:
I can do it with this PR, I already played with this idea, but didn't finish it. I'll also remove then variants of steps with dictionaries as they create unnecessary variations of API